app/views/support_interface/provider_users/_provider_user_navigation.html.erb

```diff
@@ -18,6 +18,6 @@
 <% end %>
 
 <%= render TabNavigationComponent.new(items: [
-  { name: 'Details', url: edit_support_interface_provider_user_path(@form.provider_user) },
```

Comment:
I think this partial could depend on a plain `provider_user` not a `form` with a `provider_user` property?